server/server/data_loader/textdata.py

```diff
@@ -131,8 +131,9 @@ def process_lang_dir(self, lang_dir, authors, data_dir=None, files_to_process=No
                             next_uid = None
 
                 author = self._get_author(root, lang_uid, uid)
-                author_short = self._get_author_short(author, authors)
-                author_uid = self._get_author_uid(author, authors)
+                author_data = self._get_author_data(author, authors)
```

Comment:
you can use something called tuple unpacking. so the proper code is:
`author_uid, author_short = self._get_author_data(author, authors)`